src/promptflow-tracing/promptflow/tracing/_span_enricher.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
+
+from .contracts.trace import TraceType
+
+
+class SpanEnricher:
+    def __init__(self):
+        pass
+
+    def enrich(self, span, inputs, output):
+        from ._trace import enrich_span_with_output
+
+        enrich_span_with_output(span, output)
```

Comment:
Why don't we enrich inputs here?

Reply:
Will add a todo here, enrich inputs is before calling the function, this enrich is after calling the function.